src/tests/intg/test_ldap.py

```diff
@@ -1169,3 +1170,130 @@ def test_nss_filters_cached(ldap_conn, sanity_nss_filter_cached):
 
     res, _ = call_sssd_getgrgid(0)
     assert res == NssReturnCode.NOTFOUND
+
+
+def rename_setup_no_cleanup(request, ldap_conn, cleanup_ent=None):
+    ent_list = ldap_ent.List(ldap_conn.ds_inst.base_dn)
+    ent_list.add_user("user1", 1001, 2001)
+    ent_list.add_group_bis("user1_private", 2001)
+    ent_list.add_group_bis("group1", 2015, ["user1"])
+
+    if cleanup_ent is None:
+        create_ldap_fixture(request, ldap_conn, ent_list)
+    else:
+        # Since the entries were renamed, we need to clean up
+        # the renamed entries..
+        create_ldap_fixture(request, ldap_conn, ent_list, cleanup=False)
+        request.addfinalizer(lambda: cleanup_ldap_entries(ldap_conn,
+                                                          cleanup_ent))
+
```

Comment:
I think that default cleanup should be enough  because it removes everything in object units
`("Users", "Groups", "Netgroups", "Services", "Policies")` or did I miss something?

Reply:
OK, calling `create_ldap_cleanup` with ent_list=None is an option, too